collector/agentPoller.go

```diff
@@ -187,15 +183,14 @@ func (a *Agent) getIP(stats chan<- statsd.StatsEvent) error {
 }
 
 // fetches container-level resource metrics from the agent (via /containers), emits to the framework topics (default 'metrics-<framework_id>')
-func (a *Agent) getContainerMetrics(agentState *AgentState, stats chan<- statsd.StatsEvent) ([]*AvroDatum, error) {
-	rootJSON, err := a.getJSONFromAgent("/containers", agentTestContainersFileFlag, stats)
+func (a *Agent) getContainerMetrics(agentState *AgentState) ([]*AvroDatum, error) {
+	rootJSON, err := a.getJSONFromAgent("/containers", agentTestContainersFileFlag)
 	if err != nil {
 		return nil, err
 	}
 
 	containersArray, err := rootJSON.ObjectArray()
```

Comment:
Same comment here about using `jason` lib versus a literal struct that we can unmarshal to (see below comment in `getJSONFromAgent()`).